terraform/shared/modules/https-proxy/prepare-proxy.sh

```diff
@@ -13,11 +13,12 @@ tmpdir=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
 cd "$tmpdir"
 
 # Grab a copy of the zip file for the specified ref
-curl -s -L https://github.com/GSA-TTS/cg-egress-proxy/archive/${GITREF}.zip --output local.zip 
+curl -s -L https://github.com/GSA-TTS/cg-egress-proxy/archive/${GITREF}.zip --output local.zip
 
 # Zip up just the proxy/ subdirectory for pushing
 unzip -q -u local.zip \*/proxy/\*
-zip -q -j -r ${popdir}/proxy.zip cg-egress-proxy-*/proxy
+cd "${tmpdir}/cg-egress-proxy-main/proxy/" && zip -r -o -X "${popdir}/proxy.zip" ./ > /dev/null
```

Comment:
Can't remember why I made this change. I think it makes sense, but will confirm locally and in preview

Reply:
NVM, can remove this change. Think I forgot about it after I made the `sandbox-proxy` module clone